namedtuple('results', 'output return_code exception')
